uint8 STOPPED = 0
uint8 READY  = 1
uint8 PUSHING  = 2
uint8 DEBUG  = 3

# what are we doing now?
uint8 state

---

uint8 OK    = 0
uint8 ERROR = 1

uint8 result
